src/styles/components/_speaker.scss

```diff
@@ -0,0 +1,38 @@
+.c-speaker {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  width: 300px;
+  height: 350px;
+  border: 1px solid black;
+  border-radius: 2rem;
+  overflow: hidden;
+  box-shadow: 8px 4px rgb(0, 0, 0);
+
+  .speaker_image_container {
```

Comment:
Can we stick to using BEM style for our classes? You can take a look at some of my styling classes to see how it works. You can also find resources [here](https://getbem.com/introduction/) and [here](https://css-tricks.com/bem-101/) for how and why it's useful